inc/profile_user.class.php

```diff
@@ -434,24 +446,40 @@ static function showForProfile(Profile $prof) {
          return false;
       }
 
-      $query = "SELECT `glpi_users`.*,
-                       `glpi_profiles_users`.`entities_id` AS entity,
-                       `glpi_profiles_users`.`id` AS linkID,
-                       `glpi_profiles_users`.`is_dynamic`,
-                       `glpi_profiles_users`.`is_recursive`
-                FROM `glpi_profiles_users`
-                LEFT JOIN `glpi_entities`
-                     ON (`glpi_entities`.`id`=`glpi_profiles_users`.`entities_id`)
-                LEFT JOIN `glpi_users`
-                     ON (`glpi_users`.`id` = `glpi_profiles_users`.`users_id`)
-                WHERE `glpi_profiles_users`.`profiles_id` = '$ID'
-                      AND `glpi_users`.`is_deleted` = 0 ".
-                      getEntitiesRestrictRequest("AND", "glpi_profiles_users", 'entities_id',
-                                                 $_SESSION['glpiactiveentities'], true)."
-                ORDER BY `glpi_entities`.`completename`";
-
-      $result = $DB->query($query);
-      $nb     = $DB->numrows($result);
+      $utable = User::getTable();
+      $putable = Profile_User::getTable();
+      $etable = Entity::getTable();
+      $iterator = $DB->request([
+         'SELECT DISTINCT' => "$utable.*",
```

Comment:
DISTINCT added ?

Reply:
To avoid possible problems with left joins I've encouter in my previous pull request